### ISSUE TITLE
[mihome] New Aqara mini wireless switch is not recognized

### ISSUE THREAD

**Issue by westward** [openhab/openhab2-addons#4191]
Two weeks ago I bought two new Xiaomi Aqara Mini Wireless Switch devices. They was successfully paired with a Xiaomi Gateway. Unfortunately they are not dicoverable by OpenHAB as things. 
According to instructions on the https://www.openhab.org/addons/bindings/mihome/#supporting-new-devices page, I collected the debug log.

```
tail -f /var/log/openhab2/openhab.log -f /var/log/openhab2/events.log | grep b1acn01
2018-11-03 13:44:18.582 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"click\"}"}
2018-11-03 13:44:18.591 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 13:44:28.222 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"double_click\"}"}
2018-11-03 13:44:28.231 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 13:44:32.766 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"long_click_press\"}"}
2018-11-03 13:44:32.779 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 13:44:34.212 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"long_click_release\"}"}
2018-11-03 13:44:34.220 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 13:44:44.922 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"iam\"}"}
2018-11-03 13:44:44.929 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 13:44:44.959 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"heartbeat","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"voltage\":3055}"}
2018-11-03 13:44:44.966 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
2018-11-03 14:17:54.919 [TRACE] [g.mihome.handler.XiaomiBridgeHandler] - Received message {"cmd":"report","model":"remote.b1acn01","sid":"158d00016c142f","short_id":52334,"data":"{\"status\":\"click\"}"}
2018-11-03 14:17:54.926 [DEBUG] [discovery.XiaomiItemDiscoveryService] - Unknown discovered model: remote.b1acn01
```

As you may see, there are short-click, double-click and long click press and release events. Also there are heartbeat, battery voltage status messages, and the result of short press on the device pair tiny button.

During googling the issue I found two web links that may be useful:
https://w3bsit3-dns.com/forum/index.php?showtopic=794186&st=7120
https://community.openhab.org/t/new-model-wxkg11lm-xiaomi-aqara-wireless-switch/54106

Is there any chance to update the mihome binding to include this new device?
Indeed, just ask if any other info or something else is needed.

**Comment by dimalo** [comment]
hi @westward, you did a great job getting the logs! Thanks to that I was able to add the device very quickly! Cheers 🍻

**Comment by bdvrieze** [comment]
Any hints how I can update this in OpenHAB? A full system update and Doctor Google fail to help me... And no replies in OpenHAB community either... (https://community.openhab.org/t/new-model-wxkg11lm-xiaomi-aqara-wireless-switch/54106)

**Comment by dimalo** [comment]
@bdvrieze you have to switch to the milestone version M7 or the snapshot version at the moment

**Comment by westward** [comment]
@bdvrieze I have added that Aqara switch to the OpenHAB system without any switching to any unstable/milestone/snapshot version. When I collected the logs, as it perfectly described on the MiHome binding documentation page, I found the device ID in the logs. After that I manually added a new Thing in the PaperUI Inbox and applied the device ID in the Thing properties.
After that the new device appears in the system as usual Thing.
